examples/bridge/src/bin/dealer.rs

```diff
@@ -29,7 +29,7 @@ fn main() {
         )
         .get_matches();
 
-    // Parse args
+    // Setup parsing
```

Comment:
The fix is to use this as the comment on line 13 and to use `// Parse arguments` here. Sorry for any confusion.